MAUI/Parallax-View/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: About .NET MAUI Parallax View control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Parallax View (SfParallaxView) control, its elements and more.
```

Comment:
description: Learn about the introduction of Syncfusion .NET MAUI Parallax View (SfParallaxView) control, its elements, and more.